### ISSUE TITLE
Temporary directories left behind

### ISSUE THREAD

**Issue by astronouth7303** [testem/testem#1135]
I'm seeing a bunch of empty `/tmp/testem-*` directories being left behind. They contain no files, but are still cluttering the filesystem.

**Comment by astronouth7303** [comment]
```
$ sudo find /tmp/testem-2294
/tmp/testem-2294
/tmp/testem-2294/Local Storage
```

**Comment by melvyn-sopacua** [comment]
When using Firefox launcher, full profiles are left behind, which will certainly clutter a small (memory-backed) temporary file system. In addition, it's better to use XDG_RUNTIME_DIR when defined.

**Comment by stefanpenner** [comment]
Yup, I've been noticing this as well. Although it is reasonable (on unexpected crash) to leak said files, a non exceptional exit of testem should absolutely cleanup after itself.
